x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -20,13 +21,18 @@ export default function ({ getService, getPageObjects }) {
     const targetIndexName = 'rollup-to-be';
     const rollupSourceIndexPattern = 'to-be*';
     const rollupSourceDataPrepend = 'to-be';
+
     //make sure all dates have the same concept of "now"
     const now = new Date();
     const pastDates = [
       datemath.parse('now-1d', { forceNow: now }),
       datemath.parse('now-2d', { forceNow: now }),
       datemath.parse('now-3d', { forceNow: now }),
     ];
+    before(async () => {
+      await security.testUser.setRoles(['manage_rollups_role', 'kibana_user']);
```

Comment:
We're assigning the `kibana_user` role here, but the `manage_rollups_role` is already granting a set of Kibana privileges. Do you need both roles granted for this test to work? If so, then removing the kibana privileges from the `manage_rollups_role` would eliminate the redundancy

Reply:
OK. I removed the kibana role just to have a single role added. We will see how that works.